cloud/pkg/cloudhub/handler/eventhandler.go

```diff
@@ -32,6 +34,7 @@ const (
 // constants for error message
 const (
 	MsgFormatError = "message format not correct"
+	VolumeRegExp   = `^\w[-\w.+]*/` + constants.CSIResourceTypeVolume + `/\w[-\w.+]*`
```

Comment:
You can call `regexp.Compile()` right after VolumeRegExp set, avoid repeat calling it in MatchString()

Reply:
Done. Thanks